lib/benchee/formatters/csv/general.ex

```diff
@@ -0,0 +1,63 @@
+defmodule Benchee.Formatters.CSV.General do
+  alias Benchee.{Statistics, Benchmark.Scenario}
+
+  @column_descriptors ["Name", "Input", "Iterations per Second", "Average",
+                       "Standard Deviation",
+                       "Standard Deviation Iterations Per Second",
+                       "Standard Deviation Ratio", "Median", "Minimum",
+                       "Maximum", "Sample Size"]
+  @moduledoc """
+    Functionality for converting Benchee scenarios to general csv.
+  """
+
+  @doc """
+    Adds headers to given scenarios.
+
+    ## Examples
+      iex> Benchee.Formatters.CSV.General.add_headers([])
+      [["Name", "Input", "Iterations per Second", "Average", "Standard Deviation", "Standard Deviation Iterations Per Second", "Standard Deviation Ratio", "Median", "Minimum", "Maximum", "Sample Size"]]
```

Comment:
Breaking the line here in the doc tests would help I guess :)

Reply:
sure